src/azul/service/manifest_service.py

```diff
@@ -1532,6 +1533,12 @@ def create_file(self) -> tuple[str, Optional[str]]:
         avro_pfb.write_pfb_entities(entities, pfb_schema, path)
         return path, None
 
+    def _inject_reference_handover_columns(self, field_types):
```

Comment:
Ideally, this method would be in close proximity to `_inject_reference_handover_values`. Readers are unlikely to make the connection. I certainly didn't in my first review.